site/docs/getting-started.md

```diff
@@ -17,10 +17,14 @@
 
 # Getting Started
 
-## Using Iceberg in Spark 3
-
 The latest version of Iceberg is [{{ versions.iceberg }}](./releases.md).
 
+Spark is currently the most feature-rich compute engine for Iceberg operations. 
```

Comment:
Please consider rewording: "We recommend using Spark to learn Iceberg concepts and features since it is currently the most feature-rich compute engine for Iceberg operations. However, if you prefer to start with a different engine, we ask that you consult the documentation for directions on using Iceberg with other compute engines. This documentation is located under the **Engines** tab."